project_management/tps-funding-application-20190429.md

```diff
@@ -0,0 +1,205 @@
+The book does not currently include the aspects of data science and artificial intelligence that define whether work was ethically designed and reported.
+By curating this knowledge in an accessible format, _The Turing Way_ series will support the The Turing Institute's goals of delivering world class research, training the leaders of tomorrow, and leading the public conversation.
+
+_The Turing Way_ will not replace academic journals or software packages.
+Rather it is a pedagogical opportunity.
+A translation of research outputs into curated tutorials, case studies and checklists.
+The books are Jupyter Books (https://github.com/jupyter/jupyter-book) which build and host static web content directly from markdown files or jupyter notebooks in a GitHub repository.
+They are easy to version control, edit and extend.
+The notebooks are interactive via a Binder plugin.
+
+The _Turing Way_ community is a global collaboration of research engineers, data librarians, industry professionals, and domain expert researchers at all levels of seniority.
+Anyone can contribute to the book at any time and our community manager will provide 1:1 mentorship for anyone who wants to incorporate their knowledge into the books.
+We will focus on inclusion and creativity at our four "book dash" events.
+These one day sprints allow us to bring together a diverse group to co-create the most useful resource for all.
+They will contribute their knowledge to the book, review the content for clarity across multiple domains of expertise, and create personalised pathways through the content to add value to as broad a community as possible.
+
+Metrics of success over 18 months:
+
+* 20 new chapters.
+* 100 authors (significant contributions to the books).
+* 200 contributors (any contribution to the books).
+* 1000 mailing list subscribers (currently 200).
+* 50 first pull requests.
+* 20 new contributors to other open source projects.
+
+
+### Route to impact
+
+*Max 300 words*
+
+We are committed to giving all early career data scientists the support they need to confidently publish reproducible, accurate, interpretable, and responsible research.
+We will support financial analysts, civil servants, people in third sector organisations, and domain experts who need to evaluate applications of data science and AI.
+_The Turing Way_ books will be accessible to everyone who wants to learn these skills and the content will adapt to accommodate the needs of these communities.
+
+Pathways:
+  * **Training Steering Group** - use of The Turing Way chapters in doctoral training.
+  * **Book dash events** - respond to community requests for content and build confidence in early career researchers to share their skills.
+  * **Research engineering and partnerships teams** - chapter on effectively scoping an interdisciplinary project.
+
+_The Turing Way_ will enable world class research by facilitating communication interdisciplinary teams.
```

Comment:
by facilitating communication *within* interdisciplinary teams?

Reply:
Yep 🤦‍♀ Thank you!